source/scripts/tutorial_page.js

```diff
@@ -0,0 +1,91 @@
+// Master DOManipulator v2 ------------------------------------------------------------
```

Comment:
Commenting could use a bit of cleanup, perhaps some more headers and non-inline comments for better readability